test/src/main/resources/DeploymentSpecifics.properties

```diff
@@ -106,4 +106,6 @@ jdbc.querylistener.key=260873
 MemoryLeak.active=false
 
 #needs a comma separated list with mailaddresses for the larva adapter
+#people in the .error-list are only mailed when tests fail
 errorreport.mailaddresses=
+errorreport.mailaddresses.error=
```

Comment:
I would like to suggest to name these mailaddresses:
errorreport.mailaddresses.pass
errorreport.mailaddresses.fail